src/main/java/org/broadinstitute/hellbender/utils/test/CommandLineProgramTestInterface.java

```diff
@@ -0,0 +1,94 @@
+package org.broadinstitute.hellbender.utils.test;
+
+import htsjdk.samtools.util.Log;
+import org.broadinstitute.hellbender.Main;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.logging.BunnyLog;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Utility interface for CommandLine Program testing.
```

Comment:
Could you add a comment explaining why someone would implement this?  It's hard to understand the point if you're not thinking about multiple `Main` implementations.